tests/unit/cartography/intel/aws/test_iam.py

```diff
@@ -5,10 +5,26 @@
     "Action": "*",
 }
 
+SINGLE_PRINCIPAL = {
```

Comment:
Please add a comment with a link to the AWS datatype that this is meant to copy (the `Principal` field in a policy statement).